.github/workflows/ci-pip-arch.yml

```diff
@@ -44,21 +44,21 @@ jobs:
       uses: uraimo/run-on-arch-action@v2
       with:
         arch: ${{ matrix.arch }}
-        # distro: ubuntu_rolling
-        # ubuntu 22.04 uses Python 3.10
-        distro: ubuntu22.04
+        # use the latest LTS version
+        distro: ubuntu_latest
```

Comment:
Do we want to keep distro pinned to ubuntu 24.04 rather than justr taking 'ubuntu_latest'?

Reply:
So, the `distro` setting is used by https://github.com/uraimo/run-on-arch-action and has a limited number of values - see https://github.com/uraimo/run-on-arch-action?tab=readme-ov-file#supported-platforms

So, this change switches from ubuntu 22.04 to the latest version, which is currently ubuntu 24.04. According to the documentation there is no `ubuntu-24.04` setting, which is why I picked `ubuntu-latest` as the most sensible choice here.